stats.go

```
@@ -0,0 +1,65 @@
+	"sync"
+	"time"
+
+	"github.com/manucorporat/stats"
+)
+
+var (
+	ips        = stats.New()
+	messages   = stats.New()
+	users      = stats.New()
+	mutexIps   sync.RWMutex
+	mutexUsers sync.RWMutex
```

golangci-lint (1.64.8)

var `mutexUsers` is unused

(unused)